pkgs/development/python-modules/requests-futures/default.nix

```
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, requests, lib, pythonOlder }:
```

`pythonOlder` does not seem used

```suggestion
{ buildPythonPackage, fetchPypi, requests, lib }:
```

---

Fixed both